src/app/home/home.component.ts

```diff
@@ -74,6 +74,20 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
             next: () => console.log('pinging server...'),
             error: () => this.disableEditor()
         });
+
+        this.http.get(this.pingURL).subscribe({
+            next: () => console.log('pinging server...'),
+            error: (err: HttpErrorResponse) => {
+                if (err.status === 429) {
+                    
+                    console.log('Shume kerkesa, provo me vone.');
+                    this.disableEditorToManyRequests();
+                } else {
+                    
```

Comment:
Should `disableEditor` have been called here?

Reply:
Yes, Because we need to disable the editor and show the message after the 429 code has been triggered, do you have any suggestions?